ui/src/containers/organisations.list/users/index.tsx

```diff
@@ -74,26 +75,48 @@ export default function OrganisationUsers() {
 
   const getOrganizationUser = useCallback(
     async (orgId: string) => {
-      const resp = await client?.frontierServiceListOrganizationUsers(orgId, {
-        with_roles: true,
-      });
-      const userList = resp?.data?.users || [];
-      const role_pairs = resp?.data?.role_pairs || [];
-      setOrgUsers(userList);
-      setRolePairs(role_pairs);
+      try {
+        setIsUsersLoading(true);
+        const [usersResp, invitationResp] = await Promise.all([
+          client?.frontierServiceListOrganizationUsers(orgId, {
+            with_roles: true,
+          }),
+          client?.frontierServiceListOrganizationInvitations(orgId),
+        ]);
+        const userList = usersResp?.data?.users || [];
+        const role_pairs = usersResp?.data?.role_pairs || [];
+        const invitedUsers =
+          invitationResp?.data?.invitations?.map((user) => ({
```

Comment:
We often have business logic of the kind written above where we fetch data and then filter it in some way on frontend.
Should we have helper functions for things like this? For example, a helper called `filterInvitedUsers(userList)` could do this, and we could reuse it. Since Frontier is under active development, having such helper methods would help us easily change things on frontend if something changes on the backend.

Reply:
We are not filtering users here. We are adding an extra field to invited users and merging it will users